pkg/reconciler/domainmapping/reconciler.go

```diff
@@ -67,7 +75,17 @@ func (r *Reconciler) ReconcileKind(ctx context.Context, dm *v1alpha1.DomainMappi
 
 	logger.Debugf("Mapping %s to %s/%s", url, dm.Spec.Ref.Namespace, dm.Spec.Ref.Name)
 	desired := resources.MakeIngress(dm, ingressClass)
-	_, err := r.reconcileIngress(ctx, dm, desired)
+	ingress, err := r.reconcileIngress(ctx, dm, desired)
+	if err != nil {
+		return err
+	}
+
+	if ingress.GetObjectMeta().GetGeneration() != ingress.Status.ObservedGeneration {
+		dm.Status.MarkIngressNotConfigured()
```

Comment:
I'd add a comment that it's ingress we're looking at, since it took me a moment to see that it's different object from above 😊

Reply:
this is copied from https://github.com/knative/serving/blob/master/pkg/reconciler/route/route.go#L161 which also doesn't have comment, but sure I'll add one